Makefile

```diff
@@ -0,0 +1,25 @@
+PACKAGE_VERSION = $(shell cat VERSION)
+COV_CONFIG = ".coveragerc"
+SOURCE_FOLDER = "src"
+TESTS_FOLDER = "tests"
+TESTS_PARAMS = "-p no:cacheprovider"
+
+
+.PHONY: check
+check:
+	@echo "Checking code format"
+	@black --check $(SOURCE_FOLDER)
+	@black --check $(TESTS_FOLDER)
+
+
+.PHONY: tag
+tag:
+	@echo "Tagging current version"
+	@git tag --annotate "v$(PACKAGE_VERSION)" --message "Tag v$(PACKAGE_VERSION)"
+	@git push --follow-tags
```

Comment:
Maybe just use `bump2version`? As this is using semvar it handles it quite nicely.

Reply:
`bump2version` added.

Please review, I am not that familiar with this tool.